cmd/v/help/build.txt

```diff
@@ -37,6 +37,7 @@ NB: the build flags are shared with the run command too:
       Specify the backend to use while building the executable.
       Current list of supported backends:
       * `c` (default)       - V outputs C source code which is passed to a C compiler to be compiled.
+      * `interpret`         - Same as `v interpret` to run the V program
```

Comment:
This was deliberately not here - unlike the other backends, the interpreter does not produce an intermediate result, but runs the AST directly, so only `v interpret file.v` makes sense imho.

Reply:
This for example `./v -b interpret run examples/hello_world.v` fails with:
```
Hello, World!
No such file or directory
```